cgmes/cgmes-conformity/src/test/resources/cgmes3-test-models/MicroGrid/20171002T0930Z_ENTSO-E_EQ_BD_2.xml

```diff
@@ -0,0 +1,269 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<rdf:RDF xmlns:cim="http://iec.ch/TC57/CIM100#" xmlns:eu="http://iec.ch/TC57/CIM100-European#" xmlns:md="http://iec.ch/TC57/61970-552/ModelDescription/1#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
```

Comment:
This file is going to be deprecated. It means that all the infos should be in the `EQ` file. Is it true for the test files?